dolfyn/rotate/rdi.py

```diff
@@ -148,8 +149,14 @@ def inst2earth(adcpo, reverse=False,
 
 def calc_orientmat(adcpo):
 
-    # Now calculate the rotation matrix.
+    # Calculate the orientation matrix using the raw 
+    # heading, pitch, roll values from the RDI binary file.
+
     """
+     Parameters
+    ----------
+    -adcpo : The ADP object containing the data.
```

Comment:
remove the `-` at the beginning of this line.

Reply:
Done.